src/langchain_google_alloydb_pg/async_vectorstore.py

```diff
@@ -488,6 +516,35 @@ async def asimilarity_search(
             embedding=embedding, k=k, filter=filter, **kwargs
         )
 
+    def _images_embedding_helper(self, image_uris: List[str]) -> List[List[float]]:
+        # check if `embed_image()` API is supported by the embedding service used
+        if hasattr(self.embedding_service, "embed_image"):
```

Comment:
Are there examples of child `langchain_core.embeddings::Embeddings` class where `embed_image` is implemented?

Reply:
Currently there are: